src/JoomlaBrowser.php

```diff
@@ -50,8 +50,8 @@ public function doAdministratorLogin($user = null, $password = null)
 
         $this->debug('I open Joomla Administrator Login Page');
         $I->amOnPage('/administrator/index.php');
-		$I->waitForElement(['id' => 'mod-login-username'], 60);
-		$this->debug('Fill Username Text Field');
+        $I->waitForElement(['id' => 'mod-login-username'], 60);
+        $this->debug('Fill Username Text Field');
```

Comment:
Thanks very much for your pull ^_^

Note that your pull is affecting many lines changing spaces for tabs. We are aware that we need to consolidate the code style in the project, but is something that will be done with #92.

Please refactor your pull request so it ONLY modify these lines that add the "fix for mobile environment". I mean not to commit changes like the one in previous lines 53 and 54. Since this one is a change not related with the fix that you are sending.

Thanks very much